src/features/activate-migration/hooks/useActivatePointsClaim.ts

```diff
@@ -0,0 +1,66 @@
+import axios from "axios";
+import { MerkleTree } from "merkletreejs";
+import { keccak256 } from "viem";
+import { useAccount, useQuery } from "wagmi";
+import { generateMerkleLeaf } from "../../votes/utils/merkleUtils";
+import { ACTIVATE_TREE_ID } from "../constants";
+
+export type Claim = {
+  tree: `0x${string}`;
```

Comment:
Not important for the PR, but Viem has a type called `Hash` which can be used instead of `0x${string}`